journals/apps/theming/models.py

```diff
@@ -18,7 +19,16 @@ class SiteBranding(BaseSetting):
 
     theme_name = models.CharField(max_length=128, blank=True, default='')
 
+    footer_links = JSONField(
+        verbose_name=_('Footer Links'),
+        help_text=_("JSON should be in the format: {\"Link Name\": \"link-url\"}.  For example: {\"About Page\": \"www.example.com\", \"Contact Us\": \"www.example.com\"}"),
```

Comment:
I think there was a PSA about using gettext on admin pages because they aren't escaped. Not sure if that's true for wagtail settings or not.